### PR TITLE
ng.datepicker style

### PR DESCRIPTION
**Pull request by sanlucca**
Style for the datepicker/daterangepicker.
The plugin/ngMaterial folder contains the necessary style to use Material2 components and overrides the necessary classes.

### CODE REVIEW

File: packages/ng/src/plugins/ngMaterial/_general.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import
+	'~@angular/material/theming', // Basic file for custom theme
```
Comment (lucienbertin) [inline packages/ng/src/plugins/ngMaterial/_general.scss]: is this mandatory ?

File: packages/ng/src/style/_components.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
 @import
 
-  'components/input';
+  'components/input',
+  '../plugins/ngMaterial/general';
```
Comment (lucienbertin) [inline packages/ng/src/style/_components.scss]: why do you import this here ?, in `_components` ? we may distribute components that dont need to override ng material

File: packages/ng/src/plugins/ngMaterial/_general.scss (hunk 1 above)
Comment (sanlucca) [reply packages/ng/src/plugins/ngMaterial/_general.scss]: it gives access to the `angular-material-typography` mixin which allow us to easily override fonts all over material components

File: packages/ng/src/style/_components.scss (hunk 2 above)
Comment (sanlucca) [reply packages/ng/src/style/_components.scss]: I put it there by default and forgot to double check afterwards. I have to try if we can import it directly into each component scss when needed, but I'm afraid some overrides (overlay and anything that end up into the overlay zone) might need to be imported globally.

**Pull request merged**